### ISSUE TITLE
aria-relevant missing (default) notation in values table 

### ISSUE THREAD

**Issue by cookiecrook** [w3c/aria#1265]
This change is needed to resolve the attribute reflection IDL for this enumerated value in #1261

**Comment by jnurthen** [comment]
closed by #1266
